azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/OSProfile.java

```diff
@@ -91,15 +90,58 @@ public static LinuxConfiguration create(final String disablePasswordAuthenticati
 
       @AutoValue
       public abstract static class WinRM {
+          public enum Protocol {
+
+              HTTP("http"),
+              HTTPS("https"),
+              UNRECOGNIZED("Unrecognized");
+
+              private String value;
+
+              Protocol(String value) {
+                 this.value = value;
+              }
+
+              public static Protocol fromValue(String value) {
+                 Protocol[] items = Protocol.values();
```

Comment:
Could use the same pattern as https://github.com/jclouds/jclouds-labs/blob/master/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/ComputeNode.java#L28, i.e.

    return (Protocol) GetEnumValue.fromValueOrDefault(value, Protocol.UNRECOGNIZED);